go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/aws/copilot-cli
 
-go 1.16
+go 1.17
```

Comment:
We need to upgrade all the buildspecs too!
- https://github.com/aws/copilot-cli/pull/2690/files
- https://github.com/aws/copilot-cli/pull/2687/files

Reply:
updated!